### DISCUSSION TITLE
Chromium passwords not being saved when switching between xfce and i3wm

### DISCUSSION THREAD

**Discussion by dafnisrey** [i3/i3#6052] [Q&A] [answered]
Hello.

I have used xfce for years but recently i have installed i3 to try it. I use chromium. The problem is when i close my desktopo session with xfce and login into i3wm AND i open chromium, when i come back to xfce all my chromium sessions are exited. I mean, the session on google account, instagram, facebook, all are exited. The passwords are saved in chromium so it automatically fill the password textbox but the problem is the session is exited so i must re-enter. This happens also if i login into my chromium accounts using i3wm but then i switch to xfce and back to i3.

I apprecciate any help.
Thank you.

**Comment by tycho-kirchner** [comment, marked as answer]
Take a look at chromium's command line option `--password-store`. Personally, I have the gnome keyring (which runs in Xfce per default), so you could use that via `--password-store=gnome`.

**Reply by dafnisrey** [reply]
Ok, im going to check that. Thank you.

**Reply by bermeitinger-b** [reply]
It's `--password-store=gnome-libsecret` (probably for newer versions?)
